app/src/main/java/edu/uc/it3048/mypokedex/MainActivity.kt

```diff
@@ -4,20 +4,19 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ImageButton
-import edu.uc.it3048.mypokedex.ui.main.MainFragment
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.main_activity)
 
-        // Calling the login method
+        /** Calling the login method */
```

Comment:
Ah, neither, actually.
1) KDoc/JavaDoc syntax should only be above a class, interface, or method/function; not inside a method/function.
2) Inside a function, use inline comments.  But..
3) Inline comments tend to be avoided by developers.  Code should be self-documenting, and inline comments are often seen as an indication that the code is not self-documenting.  If it is self-documenting, then inline comments are seen as redundant.  In this case, the comment is repeating the line below it, so it is redudant.

My advice: Remove the comment entirely.